app-editors/neovim/neovim-0.12.0~git.recipe

```diff
@@ -112,12 +119,14 @@ BUILD()
 		-DUSE_BUNDLED_UNIBILIUM=OFF \
 		-DUSE_BUNDLED_LUAJIT=OFF \
 		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_BINDIR=$commandBinDir \
```

Comment:
I think you need to move this line after $cmakeDirArgs, it's still installing in $prefix/bin/x86, my guess is that $cmakeDirArgs overrules -DCMAKE-INSTALL-BINDIR where it should be the other way around.
Same as in the next section.
Not sure if I get around to checking that today.